test/test_ops.jl

```diff
@@ -7,7 +7,7 @@ end
 
 @testsection "tensor ops" begin
 for T in (Float32, Float64, F64), dim in (1,2,3)
-
+println("T = $T, dim = $dim")
```

Comment:
It is there because sometimes there were time outs for this section because nothing got printed for 10 minutes. I think maybe when Travis is extra slow. So good to have something that prints maybe?